src/locale/locales/pt-BR/messages.po

```diff
@@ -7673,7 +7673,7 @@ msgstr "Versão {appVersion} {bundleInfo}"
 
 #: src/view/com/util/post-embeds/VideoEmbedInner/VideoEmbedInnerNative.tsx:180
 msgid "Video"
-msgstr ""
+msgstr "Video"
```

Comment:
it’s better to use "vídeo" with an accent in portuguese instead of "video" to match the correct spelling

Reply:
ok